content/docs/setup/kubernetes/install/multicluster/shared-vpn/index.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: VPN Connectivity
-description: Install an Istio mesh across multiple Kubernetes clusters with direct network access to remote pods.
+title: Shared control plane (single-network)
+description: Install an Istio mesh across multiple Kubernetes clusters with a shared control plane and VPN connectivity between clusters.
```

Comment:
its not just VPN connectivity - any routed pod network will do.  This could be a VPN, or a VPC, or even plain old l3 routing in an enterprise.